tests/fixtures/macos/macos.json

```diff
@@ -8,6 +8,7 @@
         "name": "Nix Store",
         "case_sensitive": false,
         "encrypt": false,
+        "enterprise_edition": true,
```

Comment:
Why do the values for enterprise_edition vary? Seems weird that it's `true` here and `false` elsewhere.

Reply:
that is definitely wack